packages/reports/tests/dummy/config/environment.js

```diff
@@ -48,7 +48,8 @@ module.exports = function(environment) {
         multipleExportFileTypes: ['pdf', 'png'],
         enabledNotifyIfData: true,
         enableContains: true,
-        enableTableEditing: true
+        enableTableEditing: true,
+        enableExport: false
```

Comment:
Do we need `enableExport` and `multipleExportFileTypes`?

Could we just have `exportFileTypes` and if it is empty or undefined export would be disabled?